inspirehep/records/utils.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""INSPIRE module that adds more fun to the platform."""
+from inspire_utils.name import ParsedName
+from inspire_utils.record import get_value, get_values_for_schema
+
+
+def get_pid_from_record_uri(record_uri):
```

Comment:
https://github.com/inspirehep/inspirehep/blob/master/inspirehep/pidstore/api/base.py#L104

Reply:
Thanks... That was leftover, PidstoreBase.get_pid_from_record_uri() is used in the code.